bin/stable/prepare_project_edition.sh

```diff
@@ -35,11 +35,11 @@ if [ -f ${DEPENDENCY_PACKAGE_DIR}/auth.json ]; then
 fi
 
 if [[ $PHP_IMAGE == *"7."* ]] || [[ $PHP_IMAGE == *"8.0"* ]] || [[ $PHP_IMAGE == *"8.3"* ]]; then
-    echo "> Running composer update"
-    docker exec install_dependencies composer update --no-scripts --ansi
-else
     echo "> Running composer install"
     docker exec install_dependencies composer install --no-scripts --ansi
```

Comment:
I believe `composer install` will be only possible on PHP 8.3, all lowers versions will have to user the `composer update` path

Reply:
Updated in https://github.com/ibexa/ci-scripts/pull/91/commits/91a87ea20f126f7eaa3e4f9d9577098929647d06. Thanks!